Analysis/analysis.r

```diff
@@ -146,17 +227,17 @@ dev.off()
 
 # change in intake
 
-obese_df %>% 
+full %>% 
+  filter(bmi_class %in% c("overweight", "obese", "morbidly obese")) %>% 
   mutate(calRed = ei/intake_start) %>% 
-  mutate(sex = paste0("Obese ", sex, "s")) %>% 
   dplyr::select(calRed, sex, wt_int, bmi_class) %>% 
   ggplot(., aes(x = calRed, fill = bmi_class)) +
   stat_density(aes(weight = wt_int), adjust = 3, lwd = 2, geom = "line", position = "identity", color = "#0000FF") +
   scale_x_continuous(labels = scales::percent) +
-  facet_grid(sex ~ .) +
+  facet_grid(bmi_class ~ sex) +
   labs(x = "kcal/day",
        color = "",
-       title = "Distribution of Difference in kcal/day: Obese",
+       title = "Distribution of Difference in kcal/day",
        subtitle = "As a % of baseline intake") +
   theme_ipsum(base_size = 15, axis_title_size = 15, base_family="Averta")
 
```

Comment:
The `bmi_distribution_50reduction.png` plot is really nice! (I'm not able to comment on the line where its saved :) )